plugins/com.google.cloud.tools.eclipse.appengine.login/src/com/google/cloud/tools/eclipse/appengine/login/ui/AccountSelector.java

```diff
@@ -73,7 +82,7 @@ public String getSelectedEmail() {
   }
 
   public int selectAccount(String email) {
-    int index = combo.indexOf(email);
+    int index = Strings.isNullOrEmpty(email) ? -1 : combo.indexOf(email);
     if (index != -1) {
```

Comment:
That's intended not to change the previously selected account if the method fails to find a new selection.